src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
-// write code here
+const employees = document.getElementsByTagName('li');
+
+function sortList(list) {
+  const array = [...list];
+
+  array.sort((first, second) =>
+    formatSalary(second.dataset.salary) - formatSalary(first.dataset.salary)
+  );
+
+  document.querySelector('ul').append(...array);
+}
+
+function formatSalary(string) {
+  let checkingArray = string.split('');
+
+  checkingArray = checkingArray.filter(element => !isNaN(element));
+
+  return Number(checkingArray.join(''));
+}
+
+function getEmployees(list) {
+  const result = [];
+
+  [...list].forEach(element =>
```

Comment:
there is no need create a copy of the array

Reply:
it is html collection, not an array, and does not have forEach method